weavetest/assert/assert.go

```diff
@@ -105,3 +105,21 @@ func FieldError(t testing.TB, err error, fieldName string, want *errors.Error) {
 
 	}
 }
+
+// IsErr is a convenient helper that checks if the errors are a match
+// and prints out the difference if not as well as failing the assertion.
+func IsErr(t testing.TB, want, got error) {
+	t.Helper()
+
+	if want == got {
+		return
+	}
+
+	if want, ok := want.(interface {
+		Is(error) bool
+	}); ok && want.Is(got) {
```

Comment:
If you want to inline it like this, please put the whole interface declaration in single line
```go
if want, ok := want.(interface { Is(error) bool }); ok && want.Is(got) {
```

Reply:
I disagree, it reads better like this cause then you can see the interface as declared.

Reply:
I hate inline interface and struct defs.
Can you just make

```go
type iser interface {
  Is(error) bool
}
```

above the function and refer to it?

`if iwant, ok := want.(iser); ok && iwant.Is(got) {`...